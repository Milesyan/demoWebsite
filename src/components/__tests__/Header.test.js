/* eslint-disable no-undef */ /*eslint-disable react/react-in-jsx-scope*/

import { Header } from '../index';

const setup = overrideProps => {
  const initProps = {
  };
  const props = { ...initProps, ...overrideProps};
  const wrapper = shallow(
    <Header {...props}/>
  );
  return { wrapper, props };
}

describe('Header snapshots', () => {

  it('should match snapshot', () => {
    const { wrapper } = setup();
    expect(wrapper).toMatchSnapshot();
  });

});