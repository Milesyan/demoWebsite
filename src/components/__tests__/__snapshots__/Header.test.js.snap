// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshots should match snapshot 1`] = `
<div
  className="container"
>
  Broccoli & Co.
</div>
`;
